src/app/components/Figure/Copyright/index.jsx

```diff
@@ -29,4 +32,15 @@ const Copyright = styled.span.attrs({
   }
 `;
 
+const Copyright = ({ children }) => (
+  <StyledCopyright>
+    <VisuallyHiddenText>Image source,</VisuallyHiddenText>
```

Comment:
As we are already passing `children` in, it might be cleaner to pass `VisuallyHiddenText` in as a child. This would allow us to continue to export the styled Copyright component, and would mean that we can handle things like translations (of `Image source`) at a container rather than component level.

What do you think?

Reply:
Yep that sounds fine. Let's just do it closer to the time and not let it hold us up now.